contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -70,6 +71,63 @@ reasonable production worthy environments:
   bypass policies that are not yet GA, but which may reasonably be enabled on a
   conformant cluster (e.g., pod security policy, non-GA scheduler annotations)
 
+### Windows & Linux Considerations
+
+As of v1.14, Windows is a stable feature, but not required in all Kubernetes
+cluster and is therefore not part of the conformance suite. Nonetheless, it's
+importatnt to verify that the behavior of Windows nodes conformance as much as
+possible. To that end, a large number of conformance tests are
+already included in Windows testing. You can see what tests have already pass by
+looking at TestGrid for results of Windows tests running on
+[Azure](https://testgrid.k8s.io/sig-windows#aks-engine-azure-windows-master) and
+[GCE](https://testgrid.k8s.io/sig-windows#gce-windows-master)). Tests may be
+scheduled for any PR with the bot command `/test pull-kubernetes-e2e-aks-engine-azure-windows`.
+
+Generally speaking, the goals are to:
+
+- Make sure tests that are already passing remain passing. If new OS-specific
+functionality is added, it should be in a new test.
+- Ensure that new tests covering Linux-specific functionality are tagged with `[LinuxOnly]` 
+(see: [Kinds of Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests),
+  - Give future reviewers a reference to an active issue or documentation clarifying why a test
```

Comment:
This line shouldn't be indented.

s/^  -/-

Reply:
thanks. fixed.